neo/Wallets/NEP5Token.py

```diff
@@ -277,6 +283,15 @@ def CrowdsaleRegister(self, wallet, register_addresses):
 
         return tx, fee, results
 
+    def Serialize(self, writer):
+        writer.WriteVarString(self.name)
```

Comment:
"""
        Serialize object.

        Args:
            writer (neo.IO.BinaryWriter):
        """

Reply:
😆 glad the docstring police are in force